j/julia/julia_v1.7.2_ubi8.5.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : julia
+# Version       : v1.7.2 (master branch, commit:bf534986350a991e4a1b29126de0342ffd76205e)
```

Comment:
`v1.7.2` should be enough. We can omit `(master branch, commit:bf534986350a991e4a1b29126de0342ffd76205e)`.